imagetest/cmd/manager/main.go

```diff
@@ -84,6 +85,14 @@ var (
 		"ubuntu-pro-1604-lts":     "projects/ubuntu-os-pro-cloud/global/images/family/ubuntu-pro-1604-lts",
 		"ubuntu-pro-1804-lts":     "projects/ubuntu-os-pro-cloud/global/images/family/ubuntu-pro-1804-lts",
 		"ubuntu-pro-2004-lts":     "projects/ubuntu-os-pro-cloud/global/images/family/ubuntu-pro-2004-lts",
+		"windows-2012-r2-core":    "projects/windows-cloud/global/images/family/windows-2012-r2-core",
```

Comment:
I'd love to see alphabetically ordered lists/lines :-)

Reply:
I can switch the -core files around on my next PR. Interestingly, the order here is just the order that came out when I did "gcloud images list"

The reason that they come out in that order is that they are ordered by image name and not by family, so
windows-server-2016-dc-core-v20220812
sorts before
windows-server-2016-dc-v20220812